infra/gcp/clusters/projects/k8s-infra-ii-sandbox/budget.tf

```diff
@@ -0,0 +1,41 @@
+/*
+
+This file contains 
+- defined budget for this project
+- notification channel when threshold is reached
+*/
+
+resource "google_billing_budget" "budget" {
+  billing_account = data.google_billing_account.account.id
+  display_name    = "Billing Budget for ${google_project.project.id}"
+
+  budget_filter {
+    projects = ["projects/${google_project.project.number}"]
+  }
+
+  amount {
+    specified_amount {
+      units = "1000"
```

Comment:
It's purely random. Not sure if there is a requirement for this.

Reply:
What are "units" ? How do we measure our usage of "units"?
:)

Reply:
`Units` represents the amount assigned to the project. It should be part of the public billing report once you start consume resources.